src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -121,16 +114,23 @@ const PlanWizard: React.FunctionComponent<IPlanWizardProps> = ({
   };
 
   return (
-    <Modal isOpen width="95%" showClose={false} hasNoBodyWrapper>
+    <PageSection variant="light">
+      <Breadcrumb className={`${spacing.mbLg} ${spacing.prLg}`}>
+        <BreadcrumbItem>
+          <Link to={`/plans`}>Migration plans</Link>
+        </BreadcrumbItem>
+        <BreadcrumbItem>Plan Wizard</BreadcrumbItem>
```

Comment:
The word "wizard" shouldn't really be shown in the UI as the name of the page, right @vconzola ? I imagine this breadcrumb bar should read "Migration plans > Create" ?

Reply:
Looks better, I've changed and aligned it to go along with above comments.

@vconzola to confirm it or use something else.